ui-components/menuitems.md

```diff
@@ -66,11 +66,14 @@ A basic menu item triggers its `onAction` function when clicked.
 | Name | Value | Requirement | Description |
 | ---- | ----- | ----------- | ----------- |
 | text | string | optional | Text to display if no icon is found. |
-| icon | string | optional | Name of the icon to be displayed. Must correspond to an icon in the [icon pack]({{site.baseurl}}/advanced/editor-icon-identifiers/). |
+| icon | string | optional | Name of the icon to be displayed. |
```

Comment:
Is there a reason we removed the `Must correspond` part here, but kept it everywhere else below?

Reply:
No idea what happened there.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1573/commits/8cae4db75453921b9ed1a40bc51653a91eecb0e1